woocommerce/google-listings-and-ads
# Drop support for WC < 6.8 (L-2), remove few workarounds

:warning:  This should be merged after WC 7.0.0 is released or at least after GLA with min WC bumped.

### Changes proposed in this Pull Request:

This PR drops support for WooCommerce < 6.8 (soon to be L-2).

And removes a few workarounds, to keep our code shorter and cleaner.

- https://github.com/woocommerce/google-listings-and-ads/pull/1441
- https://github.com/woocommerce/google-listings-and-ads/pull/1569


### Screenshots:




### Detailed test instructions:


Follow the test instructions form both PRs
1. Check out this branch.
2. Install the latest version of WC
3. Load any admin page to see no error there.
4. Go to wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard&guide=campaign-creation-success
5. Click on "Got it"
6. No errors should be displayed.
7. Redo 3-6 for WC 6.8

### Additional details:

- Probably, we could clean up much more, like update DEWPed dependencies etc. I just picked low-hanging fruits


### Changelog entry

> Remove - Support for WC < 6.8.
